src/main/java/org/reactome/release/orthopairs/Main.java

```diff
@@ -47,7 +47,7 @@ public static void main( String[] args ) throws IOException, ParseException, Ser
         String pathToSpeciesConfig = props.getProperty("pathToSpeciesConfig", "src/main/resources/Species.json");
         String pantherQfOFilename = props.getProperty("pantherQfOFilename", "QfO_Genome_Orthologs.tar.gz");
         String pantherHCOPFilename = props.getProperty("pantherHCOPFilename", "Orthologs_HCOP.tar.gz");
-
+//        String uniprotQueryBatchSize = props.getProperty("uniprotQueryBatchSize", "100");
```

Comment:
Can/should this be removed?

Reply:
I left it in as a reminder that this batch size really ought to be configurable. I just didn't have time to do it today, I wanted to get the program working.

Reply:
Okay